src/app/routes/contentRoutes.js

```diff
@@ -82,4 +82,27 @@ module.exports = (app) => {
                 }
             })
         )
+    app.post('/content/questionset/v1/copy/:id',
```

Comment:
Why did we wrote separate method for this? Can't we use the existing QuestionSet or ContentCopy methods itself, rather than writing different for each endpoint.

What is the differentiating factor to write separate method?

Reply:
@vinukumar-vs 
I see that content route is defined with wildcard and it is using `app.all`, as we have made practice not to use wild cards or `app.all` post **API whitelisting implementation**, as this is not best / recommended practice.
So, we are expecting request method type and appropriate definition for any new route being introduced in portal backend.
The changes looks good to me
CC: @rajeevsathish @princegupta1131